GenderGapsInCollegeDegrees.md

```diff
@@ -0,0 +1,160 @@
+
+## Clean up the chart
+1. Let's explore the gender gap in STEM fields, such as science, technology, engineering, and mathematics.
+2. First off, start with the historical percentage of Biology degrees for both genders.
+
+
+```python
+import pandas as pd
+import matplotlib.pyplot as plt
+#Read data
+women_degrees = pd.read_csv('percent-bachelors-degrees-women-usa.csv')
+#Visualize the gender gap of Biology degree
+fig, ax = plt.subplots()
+ax.plot(women_degrees['Year'], women_degrees['Biology'], c='red', label='Women')
+ax.plot(women_degrees['Year'], 100-women_degrees['Biology'], c='blue', label='Men')
+#Disable tick marks
+ax.tick_params(bottom="off", top="off", left="off", right="off")
+#Erase spines
+for key,spine in ax.spines.items():
+    spine.set_visible(False)
+#Display the chart
+ax.legend(loc='upper right')
+plt.show()
+```
+
+
+![png](output_1_0.png)
+
+
+## Compare across STEMs
+1. Let's compare gender gaps across 6 degrees.
+2. Clean the chart by removing ticks, setting line width, improving the lay out and ordering, and replacing the legend with annotations.
+
+
+```python
+stem = ['Engineering', 'Computer Science', 'Psychology', 'Biology', 'Physical Sciences', 'Math and Statistics']
+fig = plt.figure(figsize=(18, 3))
+for sp in range(0,6):
+    ax = fig.add_subplot(1,6,sp+1)
+    ax.plot(women_degrees['Year'], women_degrees[stem[sp]], c='red', label='Women', linewidth=3)
+    ax.plot(women_degrees['Year'], 100-women_degrees[stem[sp]], c='blue', label='Men', linewidth=3)
+    #Set spines
+    for key,spine in ax.spines.items():
+        spine.set_visible(False)
+    ax.set_xlim(1968, 2011)
+    ax.set_ylim(0,100)
+    ax.set_title(stem[sp])
+    ax.tick_params(bottom="off", top="off", left="off", right="off")
+    # Set text annotations 
+    if sp == 0:
+        ax.text(2005, 87, 'Men')
+        ax.text(2002, 8, 'Women')
+    elif sp == 5:
+        ax.text(2005, 62, 'Men')
+        ax.text(2001, 35, 'Women')
+plt.show()
+```
+
+
+![png](output_3_0.png)
+
+
+Findings: 
+1. Degrees where women dominate are psychology.
+2. Degrees where men dominate are Engineering, CS, Physical Sciences, Math and Statistics.
+3. Big gaps in gender are Engineering, CS, Psychology.
+4. Gender gaps are decreasing are Biology and Physial Sciences.
+5. Math and Statistics are balanced.
```

Comment:
More analysis here would be interesting.  In particular, diving into these would be good:

* Why female enrollment in computer science on a percentage basis increased until 1980, then decreased after
* Why psychology female enrollment increased so dramatically
* Why male percentage enrollment in physical science has slowly been trending down

This is another case where looking at the total numbers, not just percentage enrollment, would be useful.  Googling to find papers or articles on the subject could also be helpful.